Bookmark-Manager/src/main/java/com/bookmarkmanager/pojo/Bookmark.java

```diff
@@ -0,0 +1,48 @@
+package com.bookmarkmanager.pojo;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@Entity
+@Table(name = "bookmarks")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Bookmark {
+  @Id
+  @GeneratedValue
+  private UUID  id;
+  private String title;
+  private String url;
+  private UUID userId;
```

Comment:
Why not join with User table?

Reply:
In Feature I am using UserTable for Register User and Cresting JWT token for /register and /login Url. So Seprating User table and BookMark Table